MPSGameLift/Documentation/GameLift.md

```diff
@@ -0,0 +1,140 @@
+    
+    rem Copy the executables and DLLs
+    copy .\build\windows_mono\bin\profile\*.* %1
+    
+    rem Copy launch_server.cfg
+    copy .\launch_server.cfg %1
+
+    rem Copy the AWSCore files
+    copy .\build\windows_mono\bin\profile\Gems\AWSCore\*.* %1\Gems\AWSCore
+    ```
+1. Test the profile pak server and game locally
+    Run the server in headless mode using `rhi=null` and `NullRenderer` parameters; the server appears as a white screen in headless mode.
+    ```sh
+    C:\GameLiftPackageWindows\MultiplayerSample.ServerLauncher.exe --rhi=null -NullRenderer -bg_ConnectToAssetProcessor=0 -sys_PakPriority=2 -sv_terminateOnPlayerExit=true --console-command-file=launch_server.cfg
+    ```
+    
+    ```sh
+    C:\GameLiftPackageWindows\MultiplayerSample.GameLauncher.exe -bg_ConnectToAssetProcessor=0 -sys_PakPriority=2 --connect
+    ```
+
+    Note: launch_server.cfg is required because there's a bug with multiplayer when calling --loadlevel in the command-line. See https://github.com/o3de/o3de/issues/15773.
+1. Open C:\GameLiftPackageWindows\user\log\Server.log
+    You should see the "New Starbase" level loaded
+    ```
+    Entities from new root spawnable 'levels/newstarbase/newstarbase.spawnable' are ready
+    ```
+
+## Prepare for GameLift
+### Upload the build to GameLift
+Note: Builds are tied to Fleets; you may want to delete the existing build and fleet via the AWS Gamelift dashboard just so you don't accidentally reference old builds or old fleets in future steps. 
+`
+aws gamelift upload-build --operating-system WINDOWS_2016 --build-root C:\GameLiftPackageWindows\ --name MultiplayerSample --build-version v1.0 --region us-west-2
+`
+Record BuildId for the next step. Example: build-fe4a2b2c-8ae5-4757-99a6-0ff372d03512
+
+### Create Fleet
+After running this command it'll take about an hour for the fleet to activate. Check the status on the GameLift dashboard. 
+
+`
+aws gamelift create-fleet --region us-west-2 --name GameLiftO3DTest2016 --ec2-instance-type c5.large --fleet-type ON_DEMAND --build-id <BuildId> --runtime-configuration "GameSessionActivationTimeoutSeconds=300, MaxConcurrentGameSessionActivations=2, ServerProcesses=[{LaunchPath=C:\game\MultiplayerSample.ServerLauncher.exe, Parameters= --rhi=null -sys_PakPriority=2 -NullRenderer -sv_terminateOnPlayerExit=true -bg_ConnectToAssetProcessor=0 --sv_gameLiftEnabled=true --sv_dedicated_host_onstartup=false --console-command-file=launch_server.cfg, ConcurrentExecutions=1}]" --ec2-inbound-permissions "FromPort=33450,ToPort=34449,IpRange=0.0.0.0/0,Protocol=UDP"
+`
+---
+**NOTE**
+
+The ec2-instance-type and fleet-type determines the kind of AWS resources used; your AWS account may incur costs.
+
+---
+
+Record the FleetId for the next step. Example: fleet-1a49fc3e-892a-40fc-b2e9-aa7e11983182
+
+### Create and Join Game Session
+`
+aws gamelift create-game-session --region us-west-2 --fleet-id <FleetId> --name foogamesession1 --maximum-player-session-count 3
+`
```

Comment:
repeat: This should be triple backticks to format the multiline code snippet correctly.
````suggestion
```sh
aws gamelift create-game-session --region us-west-2 --fleet-id <FleetId> --name foogamesession1 --maximum-player-session-count 3
```
````

Reply:
Did a markdown pass to fix ALL THE THINGS! Thank you!